app/assets/javascripts/application.js

```diff
@@ -13,4 +13,12 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
-//= require_tree .
+//= require_tree
+= require bootstrap-dropdown
+= require bootstrap
+
+// jQuery(function($) {
+// $("tr[data-link]").click(function() {
+// window.location = this.dataset.link
+// });
+// })
```

Comment:
Javascript isn't my strong suit so this is a legitimate question.

Is this commented out code doing anything?

If not, a good time to delete extraneous comments is when you do one last read of your code before submitting your pull request.